src/store/modules/sitestatus/index.js

```diff
@@ -403,6 +416,26 @@ const actions = {
     }
   },
 
+  getLatestOwmReport ({ commit, rootState, rootGetters, state }) {
+    const wema_name = rootGetters['site_config/wema_name']
+    if (wema_name) {
+      const owmReportObj = state.owmReport.find(owmReport => owmReport.wema_name === wema_name)
+      // request and store a new report if not cached or cached more than 1 hour ago
+      if (owmReportObj == undefined || (Date.now() - owmReportObj.timestamp) > (1000 * 60 * 60)) {
```

Comment:
I see what you're doing here, but we generally look at 1000, 60, and 60 here as magic numbers since at a glance they're not super meaningful. It looks like we have the moment library installed in the app, so a cleaner way to do this could be to use that.

https://stackoverflow.com/a/42529483